extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -39,6 +39,7 @@ object KinesisUtils {
    * @param ssc    StreamingContext object
    * @param streamName   Kinesis stream name
    * @param endpointUrl  Url of Kinesis service (e.g., https://kinesis.us-east-1.amazonaws.com)
+   * @param regionName   Region name to indicate the location of the Amazon Kinesis service
```

Comment:
we may want to consider making a default for this to maintain backward compatibility.

the problem is that the Scala and Java createStream() methods in this helper class will conflict if you use defaults for this.  i had the same issue with initialPositionInStream as well as storageLevel which is why they're not defaults.  

not sure we can do much about it without changing the names of the methods to createScalaStream() and createJavaStream() or equivalent.